### ISSUE TITLE
5.3.1: AI enemies deal no damage

### ISSUE THREAD

**Issue by Blocker226** [FlansMods/FlansMod#847]
Survivors have really horrible accuracy with projectile weapons, and survivors that use bullet weapons don't hit players at point blank, tested on single and multiplayer with just the mod installed. You can't die in the apocalypse from enemy fire except when its incendiary or explosive rounds being used.

**Comment by Ainihildo** [comment]
Hi Blocker226, 
I am having the same difficulty as you did. The survivors in my world, also 5.3.1, can't hit the player. I've been trying for a long time to find a fix, and came across this post. Based upon your edit, I came to the conclusion you perhaps found some sort of a solution? If that is so, I'd really like to know it. 
Thanks

**Comment by Blocker226** [comment]
Hi, thanks for bringing it up again, no, I haven't found any solution other than reverting to a old version. Correcting the edit now.

**Comment by Ainihildo** [comment]
Thanks

**Comment by blooditor** [comment]
Hey, I found out it's a programming fault. The bullet speed is 0 (You can see the bullet dropping from the NPC slowly). 
I fixed it myself with the source code by setting the bulletSpeed to 5.0f in shoot() in EntityFlansModShooter... We just have to wait for a fix here.

**Comment by Ainihildo** [comment]
Hello, thank you so much! Could you tell me the path to get to the source code for that fix, I'd like to use it as well. Thanks again!

**Comment by mgdgfbhevdyeh** [comment]
@jamioflan Hope FIx,pls.

**Comment by Blocker226** [comment]
@blooditor If you could, please attempt to submit a pull request, so that the changes can be committed quicker too.

**Comment by blooditor** [comment]
I don't really get how to make pull requests... I've made a repo with the bug fixed: https://github.com/blooditor/Flans/tree/1.8

**Comment by Ainihildo** [comment]
Okay guys, I'm so sorry, I feel like a royal fool right now, but has anything been done yet? I saw that blooditor made a repo with the bug fix, and that jamioflan edited it, but I really don't understand how this whole thing about pull requests and the such work. I'm not complaining or anything, I just really don't understand how it all works, and so if a fix has been made that I could be using right now, I have no idea how or where. If the AI shooter hasn't been fixed, that fine. What this is all about is that if it has been fixed, I am illiterate enough in the world of programming and Github to not know it.
Sorry for using your time, and I'd appreciate any help. Also please forgive me if this is considered necroposting.

**Comment by Blocker226** [comment]
Nope. No fix. Only way is to take @blooditor's source code, and compile it yourself using Java development tools. Else, we have to wait for jamioflan to come back from the dead/vacation before we'll ever see this working again.

**Comment by Ainihildo** [comment]
Yah, I was 90% sure that was the case, but my head told me I'd want to kill myself if I went a year not knowing there was a fix just because I was too prideful to ask. Guess I'll look into a guide on how to compile mods from code. Thanks for letting me know.
